FitNesseRoot/PlugIns/content.txt

```diff
@@ -0,0 +1,16 @@
+!1 Get more out of Fitnesse
+!3 Add Plugins to your installation and extend Fitnesse with additional great features.
+ * Suport for other programming languages
+ * Additional table types to express your tests and requirements in other ways
+ * Additional Wiki Markup commands 
+ * Fixture frameworks you can use 
+ * and more ...
+
+See [[All Available Plugins][http://fitnesse.org/PlugIns?getPage&FitnesseVersion=${FITNESSE_VERSION}]] for a list of all available extensions.
+
+Read [[Writing Plugins][FitNesse.UserGuide.AdministeringFitNesse.WritingPlugins]] to understand how to install them or write your ownn.
+
+
+Below is the list of all installed plugins in this system:
```

Comment:
So true. 
The hope is that once we have a common place for documentation defined that each plugin developer will add something. 
Last but not least you can control it and not allow a plugin to be added to FitNesse.org which has no documentation at this place.